.eslintrc

```diff
@@ -4,7 +4,7 @@
     {
       "files": ["**/*.{mjs,js,jsx,ts,tsx}"],
       "extends": ["plugin:prettier/recommended"],
-      "env": { "node": true, "es6": true }
+      "env": { "jest": true, "node": true, "es6": true }
```

Comment:
Hmm, no, we shouldn't add jest env to our default env. It makes sense to to make a jest config section for test stuff. That's pretty much what we did here https://github.com/nodejs/nodejs.dev/blob/main/.eslintrc.js#L49

Reply:
Yep good idea